From: Jan Pokorny <pokorny_jan@seznam.cz>
Date: Sat, 16 Apr 2011 13:21:17 +0200
Subject: [PATCH] parse.c: "if(" -> "if (" adjustment


Signed-off-by: Jan Pokorny <pokorny_jan@seznam.cz>
---
 parse.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/parse.c b/parse.c
index 537055f..9e79ef0 100644
--- a/parse.c
+++ b/parse.c
@@ -2235,9 +2235,9 @@ static struct token *parse_context_statement(struct token *token, struct stateme
 {
 	stmt->type = STMT_CONTEXT;
 	token = parse_expression(token->next, &stmt->expression);
-	if(stmt->expression->type == EXPR_PREOP
-	   && stmt->expression->op == '('
-	   && stmt->expression->unop->type == EXPR_COMMA) {
+	if (stmt->expression->type == EXPR_PREOP
+	    && stmt->expression->op == '('
+	    && stmt->expression->unop->type == EXPR_COMMA) {
 		struct expression *expr;
 		expr = stmt->expression->unop;
 		stmt->context = expr->left;
-- 
1.7.1

